dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/WSRdbManagedConnectionImpl.java

```diff
@@ -356,6 +356,12 @@ public boolean isMCStale() {
      * flag to keep track of isolation level switching on connection support or not
      **/
     boolean supportIsolvlSwitching = false; 
+    
+    /**
+     * flag if dsConfig is set to isolation level 'NONE (0)'
+     * Used to ensure we use the drivers isolation level
+     */
+    boolean useDriverIsoLvl = false;
```

Comment:
Rename variable so it is more obvious that it is being used for transaction_none (I imagine the name is leftover from when we were using the JDBC driver default).

Reply:
I don't think we need this variable at all.  Just check dsConfig.isolationLevel == TRANSACTION_NONE in the one or two places where this information is needed.  It will be much more straightforward.